tck/src/main/java/ee/jakarta/tck/concurrent/spec/Platform/virtual/VirtualThreadServlet.java

```diff
@@ -300,10 +308,19 @@ public void testPlatformThreadFactory() throws Exception {
         
         assertNotNull(platfromThreadFactoryAnno);
         assertNotNull(platfromThreadFactoryDD);
+        
+        Thread annoThread = platfromThreadFactoryAnno.newThread(NOOP_RUNNABLE);
```

Comment:
I understand `platfrom` is the existing, not modified name used of local variable, but would it be doable to correct it in this PR (to `platform`)?

Reply:
Thanks for spotting that.  Fixing.